string mesh_file
---
bool success
